src/OrchardCore.Modules/OrchardCore.DataProtection.Azure/Startup.cs

```diff
@@ -39,6 +39,8 @@ public override void ConfigureServices(IServiceCollection services)
                         options.ContainerName,
                         options.BlobName);
                 });
+
+            services.AddSingleton<IModularTenantEvents, BlobModularTenantEvents>();
```

Comment:
Why singleton? I think transient is enough

Reply:
Totally agree. Transient is enough here.

Reply:
All other `IModularTenantEvents` implementations are registered as scoped though. That's the safest default in any case, and judging from how `ShellScope` uses these, also the appropriate one.

Reply:
Changed to scoped lifetime.